#!/usr/bin/env python3

from dynamic_reconfigure.parameter_generator_catkin import *

PACKAGE = 'tuw_trinamic_iwos_revolute_controller'

gen = ParameterGenerator()

 gen.add('reverse_left_wheel', bool_t, 0, 'reverse left wheel direction',  False)
 gen.add('reverse_right_wheel', bool_t, 0, 'reverse right wheel direction',  False)
 gen.add('swap_wheels', bool_t, 0, 'swap wheels', False)

exit(gen.generate(PACKAGE, 'tuw_trinamic_iwos_revolute_controller', 'RevoluteNode'))